src/context/LocaleProvider/LocaleProvider.tsx

```diff
@@ -14,7 +12,7 @@ const LocaleProvider = ({ children, lang }: LocaleProviderProps) => {
     const locale = locales.find((l) => {
       return l.id === localLang;
     });
-    locale?.id === "en" ? dayjs.locale({ ...en }) : dayjs.locale({ ...pl });
+    locale && dayjs.locale(locale?.id);
```

Comment:
I think this would be a better way to load dayjs locale to avoid stacking if statements in the future, like I have seen on other pr's. Let me know if you want me to revert this back